src/cycax/cycad/engines/cycax_part_freecad.py

```diff
@@ -418,22 +450,26 @@ def build(self, in_name: Path):
         FreeCADGui.activeDocument().activeView().viewTop()
         FreeCADGui.SendMsgToActiveView("ViewFit")
 
-        filepath = f"{self._base_path}/{name}/{name}"
-        doc.saveCopy(str(f"{filepath}.FCStd"))
-        self.render_to_png(Path(filepath))
-        self.render_to_dxf(doc, Path(filepath))
-        self.render_to_stl(doc, Path(filepath))
+        self.filepath = f"{self._base_path}/{name}/{name}"
+        doc.saveCopy(str(f"{self.filepath}.FCStd"))
+        for out_choice in outformats.lower().split(","):
+            ftype, fview = out_choice.split(":") if ":" in out_choice else (out_choice, None)
+            out_format = ftype.upper()
```

Comment:
`out_format = ftype.upper().strip()` if there are spaces in the CSV `split(",")` on line 455 would have preserved them